pom.xml

```diff
@@ -0,0 +1,62 @@
+<!--Used to obtain ojdbc8 and orai18n files that are needed for client-->
```

Comment:
Maybe the pom file should also exist in .travis, as it's only used for CI?
I'd like to avoid confusion around the project using maven.
The utPLSQL project is not a maven project and the existence of `pom.xml` in project root is suggesting that it is.